shared/mortgage.js

```diff
@@ -90,6 +91,7 @@ export function getActiveMortgageByBorrower(
  */
 
 export function getMortgageOutstandingAmount(mortgage) {
+  const oneHour = 60 * 60 // used to ensure the total payment of the mortgage
```

Comment:
If you're already naming the constant `oneHour`, I think it's safe to assign it to `3600` instead of `60 * 60`